### PR TITLE
Landing frontend testing

### PR DESCRIPTION
**Pull request by Prabhat-IIT**
Added description regarding versions of Python and pip required to install coala at [Home Page](https://coala.io/#/home)
[#4973](https://github.com/coala/coala/issues/4973)

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: partials/tabs/home.html [hunk 1]
```diff
@@ -33,8 +33,14 @@ <h5 class="light">simply install coala</h5>
                     </p>
                 </div>
                 <div class="col m6 s12">
+                    <p>In order to use coala, you need Python installed. In order to do so, you should install Python >= 3.4
```
Comment (nalinbhardwaj) [inline partials/tabs/home.html]: I recommend merging these lines and condensing the sentences a bit for brevity and adding a link to pip installation guide as well, therefore making it something like:
> In order to use coala, you should have [Python >= 3.4](https://www.python.org/downloads/) and [pip >= 8](https://pip.pypa.io/en/stable/installing/) installed.

File: partials/tabs/home.html [hunk 2]
```diff
@@ -33,8 +33,14 @@ <h5 class="light">simply install coala</h5>
...
+                        <a href="https://www.python.org/downloads/">here</a>
+                    </p>
+                    <p>Also make sure you have pip >= 8 installed as older versions might prevent coala from being installed properly.</p>
+                    <p>Now simply install coala by:</p>
                     <pre class="card card-content">$ pip3 install coala-bears</pre>
                     <p>You can append <code>--pre</code> to that command to get the latest prerelease straight from master!</p>
+
```
Comment (nalinbhardwaj) [inline partials/tabs/home.html]: unrelated change, please undo

**Comment by Prabhat-IIT** [comment]
I've squashed the commits but still I don't know why my commit is not following [commit guidelines](https://coala.io/commit)

**Comment by nalinbhardwaj** [comment]
@Prabhat-IIT : The shortlog needs to be under 50 characters. And a newline between body and issue reference. Also, I think you should "Closes", not "Fixes"

**Comment by Prabhat-IIT** [comment]
Done :+1:

**Comment by nalinbhardwaj** [comment]
I think you still have a tab before the body and reference? Add a newline between them as well

**Comment by Prabhat-IIT** [comment]
Nopes @nalinbhardwaj  It has `space` instead of `tab`. Moreover, are you talking about adding a new line between the two reference?

**Comment by nalinbhardwaj** [comment]
@Prabhat-IIT : yep, you shouldn't add space or anything, just normal paragraphs. And I'm talking about a newline between "It helps users/newcomers to easily install coala without any version errors." and "Closes coala/coala#4973"

**Comment by Prabhat-IIT** [comment]
Done @nalinbhardwaj

File: partials/tabs/home.html [hunk 3]
```diff
@@ -33,6 +33,11 @@ <h5 class="light">simply install coala</h5>
                     </p>
                 </div>
                 <div class="col m6 s12">
+                    <p>In order to use coala, you should have
+                        <a href="https://www.python.org/downloads/">Python >= 3.4</a> and
+                        <a href="https://pip.pypa.io/en/stable/installing/">pip >= 8</a> Installed
```
Comment (newbazz) [inline partials/tabs/home.html]: Use `&lteq` instead of `>=`, at both the places ;)

Comment (nalinbhardwaj) [reply partials/tabs/home.html]: Actually, `&gteq` 😛

Comment (newbazz) [reply partials/tabs/home.html]: yea, my bad :P

Comment (Prabhat-IIT) [reply partials/tabs/home.html]: I think it should be &ge. &gteq is showing `>eq`

Comment (newbazz) [reply partials/tabs/home.html]: google it ;p
